### ISSUE TITLE
Bug: IDE removing wire import line

### ISSUE THREAD

**Issue by dwdzdc** [rdeepc/go-wire-util-goland-plugin#3]
So I've installed the plugin recently and wanted to test it out. It seems like when it is active, upon file save the IDE removes from import line `"github.com/google/wire"`, from `wire.go` file, and this in turn causes wire command to return error: 
```
wire: /home/dawidd/projects/di/wire.go:13:2: undeclared name: wire
wire: generate failed
```
The file looks as follows. It's the 9th line that disappears upon save: 
```
//go:build wireinject
// +build wireinject

package di

import (
	"gin_tutorial/data/repositories"
	"gin_tutorial/server/controllers"
	"github.com/google/wire"
	"github.com/sirupsen/logrus"
)

func InitializeTextController() controllers.TextController {
	wire.Build(controllers.NewTextController, logrus.New, repositories.NewTextRepository)
	return *new(controllers.TextController)
}

```
 This is not the case if the plugin is disabled or not installed. Let me know if I should provide you with any additional detail.

**Comment by rdeepc** [comment]
Hi @Gumkle,

Thanks for using the plugin and providing feedback, I could reproduce the bug using the sample code you provided.
I have fixed the bug and merged the changes. Released version 0.0.8

Thanks again.

**Comment by dwdzdc** [comment]
Hi, I can confirm this has been resolved. Thank you
